plugin/web/extensions/sidebar/core/components/hooks.ts

```diff
@@ -90,11 +90,20 @@ export default () => {
     postMsg({ action: "init" }); // Needed to trigger sending initialization data to sidebar
   }, []);
 
+  // getNameFromPath("xxx/yyy/zzz") -> zzz
+  // getNameFromPath("xxxyyyzzz") -> xxxyyyzzz
+  const getNameFromPath = (path: string) => {
+    return path.split("/").slice(-1)[0];
+  };
+
   useEffect(() => {
     const catalogBaseUrl = catalogURL || backendURL;
     if (catalogBaseUrl) {
       getDataCatalog(catalogBaseUrl, catalogProjectName).then(res => {
-        setCatalog(res);
+        const catalog = res.map(item => {
+          return { ...item, name: getNameFromPath(item.name) };
+        });
```

Comment:
The catalog is very large, so please either add the getNameFromPath inside the getDataCatalog. I know that code is a bit complex, so if you have issues let me know and I'll try to help

Reply:
Done!